neard/src/genesis_validate.rs

```diff
@@ -63,6 +64,18 @@ pub fn validate_genesis(genesis: &Genesis) {
             account_id
         );
     }
+    assert!(
+        genesis.config.online_max_threshold > genesis.config.online_min_threshold,
+        "Online max threshold smaller than min threshold"
+    );
+    assert!(
+        genesis.config.online_max_threshold <= Rational::from_integer(1),
+        "Online max threshold must be less or equal than 1"
+    );
+    assert!(
+        genesis.config.gas_price_adjustment_rate < Rational::from_integer(1),
+        "Gas price adjustment rate must be less than 1"
+    );
```

Comment:
should also assert that neither numerator nor denominator can be too large. Otherwise there can be overflow in reward calculation.